fluentui_notification/src/main/java/com/microsoft/fluentui/tokenized/notification/CardNudge.kt

```diff
@@ -39,13 +39,13 @@ import com.microsoft.fluentui.tokenized.segmentedcontrols.PillMetaData
 import kotlin.math.roundToInt
 
 // TAGS FOR TESTING
-private const val CARD_NUDGE = "CardNudge"
-private const val ICON = "ICON"
-private const val ACCENT_ICON = "Accent Icon"
-private const val ACCENT_TEXT = "Accent Text"
-private const val SUBTITLE = "Subtitle"
-private const val ACTION_BUTTON = "Action Button"
-private const val DISMISS_BUTTON = "Dismiss Button"
+const val CARD_NUDGE_CARD_NUDGE = "Card Nudge"
+const val CARD_NUDGE_ICON = "Icon"
```

Comment:
Add "CardNugde"  as prefix in value to avoid name classes while finding items in UI.